test/test_ops.py

```diff
@@ -1113,277 +1111,48 @@ def test_bbox_convert_jit(self):
         torch.testing.assert_close(scripted_cxcywh, box_cxcywh, rtol=0.0, atol=TOLERANCE)
 
 
-class BoxTestBase(ABC):
-    @abstractmethod
-    def _target_fn(self) -> Tuple[bool, Callable]:
-        pass
+def area_check(box, expected, tolerance=1e-4):
+    out = ops.box_area(box)
+    torch.testing.assert_close(out, expected, rtol=0.0, check_dtype=False, atol=tolerance)
 
-    def _perform_box_operation(self, box: Tensor, run_as_script: bool = False) -> Tensor:
-        is_binary_fn = self._target_fn()[0]
-        target_fn = self._target_fn()[1]
-        box_operation = torch.jit.script(target_fn) if run_as_script else target_fn
-        return box_operation(box, box) if is_binary_fn else box_operation(box)
 
-    def _run_test(self, test_input: List, dtypes: List[torch.dtype], tolerance: float, expected: List) -> None:
-        def assert_close(box: Tensor, expected: Tensor, tolerance):
-            out = self._perform_box_operation(box)
-            torch.testing.assert_close(out, expected, rtol=0.0, check_dtype=False, atol=tolerance)
-
-        for dtype in dtypes:
-            actual_box = torch.tensor(test_input, dtype=dtype)
-            expected_box = torch.tensor(expected)
-            assert_close(actual_box, expected_box, tolerance)
-
-    def _run_jit_test(self, test_input: List) -> None:
-        box_tensor = torch.tensor(test_input, dtype=torch.float)
-        expected = self._perform_box_operation(box_tensor, True)
-        scripted_area = self._perform_box_operation(box_tensor, True)
-        torch.testing.assert_close(scripted_area, expected, rtol=0.0, atol=1e-3)
+class TestBoxArea:
+    @pytest.mark.parametrize("dtype", [torch.int8, torch.int16, torch.int32, torch.int64])
+    def test_int_boxes(self, dtype):
+        # Check for int boxes
```

Comment:
I don't think that comment is necessary as the the test name is expressive enough.

```suggestion
```